examples/localvideosnapshot/public/index.html

```diff
@@ -39,7 +39,8 @@ <h4 class="card-title">Local Video</h4>
         <div class="card">
           <div class="card-block">
             <h4 class="card-title">Snapshot</h4>
-            <canvas id="snapshot" width="320" height="240"></canvas>
+            <canvas class="snapshot-canvas hidden" width="320" height="240"></canvas>
```

Comment:
We should not hardcode the height and width here or else the canvas image will be downscaled. Instead, set the height and width in JS to match the mediastream's dimensions, then try positioning/resizing the element using css.

Reply:
This introduced some of the styling on safari to look funky. Going to fiddle with the CSS to make it look nice! Thanks Charlie!!!